controllers/quote.go

```diff
@@ -54,6 +55,7 @@ func GetQuotes(ctx *gin.Context) {
 	}
 
 	if err != nil {
+		slog.Error("Something went wrong", "error", err)
```

Comment:
we can remove this since the errors caused by querying the DB have already been logged